provers/blevm/blevm/src/main.rs

```diff
@@ -47,12 +50,8 @@ pub fn main() {
         .unwrap();
     println!("cycle-tracker-end: verify data hash");
 
-    println!("cycle-tracker-start: serializing EVM block");
-    let block_bytes = bincode::serialize(&block).unwrap();
-    println!("cycle-tracker-end: serializing EVM block");
-
     println!("cycle-tracker-start: creating Blob");
-    // Convert the EVM block into a Celestia blob.
+    // Convert the roll-up block into a Celestia blob.
```

Comment:
what's the difference between EVM block and roll-up block?

Reply:
The roll-up block embeds the EVM block.